src/main/java/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import model.Fruit;
+import service.ParserService;
+
+public class ParserServiceImpl implements ParserService {
+    private static final String COMA = ",";
+    private static final int FIRST_POSITION_IN_LINE = 0;
+    private static final int SECOND_POSITION_IN_LINE = 1;
+    private static final int THIRD_POSITION_IN_LINE = 2;
+
+    @Override
+    public Fruit getFruitFromCsvRow(String line) {
```

Comment:
let's refactor this method to take `List<String>` (that was give from `ReaderService`) and return `List<Fruit>`.